Cargo.toml

```diff
@@ -1,15 +1,15 @@
 [package]
-name = "term_grid"
+name = "uutils_term_grid"
 description = "Library for formatting strings into a grid layout"
```

Comment:
maybe add that it is a fork from term-grid ?

Reply:
Sure!